Lib/test/test_generators.py

```diff
@@ -9,6 +9,35 @@
 
 from test import support
 
+_testcapi = support.import_module('_testcapi')
```

Comment:
This broke the specs on other implementation, which don't have a `_testcapi` module.
It should always be guarded by `try: except ImportError` and skip the spec with the `unittest.skipIf` decorator.

Reply:
Please open a new bug. It seems like there are other bugs, like test_float.py:

```
    @support.requires_IEEE_754
    def test_serialized_float_rounding(self):
        from _testcapi import FLT_MAX
        ...
```

Reply:
Yeah, a new bug is the way to go. And possibly the developers guide should be updated to say whatever the policy is? (What other implementations are you thinking of? I'm a little wary about just skipping tests – this issue this test is checking for can easily exist on other implementations too...)

Reply:
Thanks, I'll create a new bug.

@njsmith I'm working on a java implementation which is based on jython. Your concern is valid, the issue might exist as well in other implementations, that's why I didn't suggest `@support.cpython_only`, but since the helper method from the `_testcapi` is also new, it should still be guarded by `skipIf`, or else the whole test crashes.

Reply:
@vstinner just check the one you posted is fine, because it's guarded by `@support.requires_IEEE_754`, and `test.support` is implemented in python.